### PR TITLE
Added password strength checks as in electron

### LINKED ISSUE #2542: Account password complexity when using golemcli
I believe I remember the GUI not allowing to use a simple password. 
Should probably implement similar logic for golemcli for security.

### PR DESCRIPTION
**Pull request by maaktweluit**
Resolves #2542

### CODE REVIEW

**Comment by maaktweluit** [comment]
@kascheri12 You are also invited to review, can not add you in the list

File: golem/interface/client/account.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 import getpass
+from zxcvbn import zxcvbn
 
 from ethereum.utils import denoms
 
 from golem.core.deferred import sync_wait
 from golem.interface.command import command, group
 
+MIN_LENGTH = 5
+MIN_SCORE = 2
```
Comment (Wiezzel) [inline golem/interface/client/account.py]: I'm not sure if score 2 counts as strong password.
```python
>>> zxcvbn('golem.pass', user_inputs=['Golem', 'adam'])['score']
2
```

File: requirements.txt [hunk 2]
```diff
@@ -57,3 +57,4 @@ raven
 Golem-Messages==1.17.2
 -e git://github.com/golemfactory/golem-smart-contracts-interface.git@e1b1f4cafeaf73035effaad501d3b04770044ce6#egg=golem_sci
 html2text==2018.1.9
+zxcvbn-python
```
Comment (Wiezzel) [inline requirements.txt]: No package version specified?

File: golem/interface/client/account.py (hunk 1 above)
Comment (maaktweluit) [reply golem/interface/client/account.py]: whow, i fully agree with you.

Wanted to match settings chosen in electron:
https://github.com/golemfactory/golem-electron/blob/039cc0a8e95bf7a3b5c1fd7676cebb842fb9ddaa/src/components/onboarding/steps/Register.js#L143

Will check if this was thought of yet and update ( both if needed ) accordingly

File: requirements.txt (hunk 2 above)
Comment (maaktweluit) [reply requirements.txt]: all versions will be locked in #2448 

Currently ( before freeze ) we only lock versions when it breaks.

Comment (Wiezzel) [reply requirements.txt]: Alright, understood.

File: golem/interface/client/account.py [hunk 3]
```diff
@@ -59,6 +63,19 @@ def unlock(self) -> str:  # pylint: disable=no-self-use
...
+            if len(pswd) < MIN_LENGTH:
+                return "Password is too short, minimum is 5"
+
+            # Check password score, same library and settings used on electron
+            account_name = getpass.getuser() or ''
+            result = zxcvbn(pswd, user_inputs=['Golem', account_name])
+            print(result['score'])
+            if result['score'] < MIN_SCORE:
+                return "Password is not strong enough. " \
+                    "Please use capitals, numbers and special characters."
```
Comment (jiivan) [inline golem/interface/client/account.py]: Maybe we could use `result['feedback']['suggestions']` here?

**Review by jiivan — APPROVED** [review]
LGTM. One suggestion

And probably some basic tests would be a good idea.

**Comment by kascheri12** [comment]
@maaktweluit Thank you for the invite. LGTM; this solves my issue.

File: golem/interface/client/account.py (hunk 1 above)
Comment (maaktweluit) [reply golem/interface/client/account.py]: Made issue with this comment, so we can round off this PR :) #2679

File: golem/interface/client/account.py (hunk 3 above)
Comment (maaktweluit) [reply golem/interface/client/account.py]: Good idea! Does need some testing / UX love, so made an issue #2678

**Comment by maaktweluit** [comment]
@kascheri12 @jiivan  @Wiezzel Thanks for the reviews!

Will add some tests for this new code and fix the merge conflict

**Pull request merged**